### PR TITLE
Introduce support for `helm install`

### PR DESCRIPTION
**Pull request by jfilipcz**
Add support for:
* Helm install sub-command
* Timeout parameter for helm install

#### What does this PR do?
This PR introduced a new task for the purpose of providing support for `helm install` command.

#### How should this be tested?
Run it against inventory simliar to this one:
```
- object: Helm Repos
  content:
      - name: Install Helm Chart 
        helm:
          name: "ABC"
          timeout: "15m"
```

#### Is there a relevant Issue open for this?
N/A

#### Who would you like to review this?
cc: @redhat-cop/openshift-applier

### CODE REVIEW

**Review by oybed — CHANGES_REQUESTED** [review]
Cleaner solution is to just replace `helm template` on line 33(34) and add `helm_timeout_option` to the parameters - the rest looks the same

**Comment by jfilipcz** [comment]
Definitely, I've replaced `helm template` with `helm install` and remove `oc apply` part as it's no longer needed

**Comment by oybed** [comment]
ha - no no -  we need to support the `template` action. Sorry, should have been more clear ... replace `helm template` with `helm  {{ helm.command }}`

**Comment by oybed** [comment]
Also, to be consistent with other parts of the `openshift-applier` inventory, use the word `action` instead of `command`

File: roles/openshift-applier/tasks/process-helm.yml [hunk 1]
```diff
@@ -28,24 +29,19 @@
     helm_values_option: "{{ helm_values_option }} --values '{{ item }}'"
   loop: "{{ helm.values_param | default([]) }}"
 
-- name: "{{ oc_action | capitalize }} OpenShift objects based on helm template for '{{ entry.object }} : {{ content.name | default(helm.name) }}'"
+- name: "Helm install chart to target Openshift cluster: '{{ entry.object }} : {{ content.name | default(helm.name) }}'"
   shell: >
-    helm template \
+    helm install \
```
Comment (oybed) [inline roles/openshift-applier/tasks/process-helm.yml:34]: replace with `{{ helm.action }}`

File: roles/openshift-applier/tasks/process-helm.yml [hunk 2]
```diff
@@ -28,24 +29,19 @@
...
        {{ helm.name }} {{ helm.chart }} \
        {{ helm_version_option }} \
        {{ helm_namespace_option }} \
+       {{ helm_timeout_option }} \
        {{ helm_set_option }} \
        {{ helm_values_option }} \
        {{ helm.flags|d('') }} \
-       | \
```
Comment (oybed) [inline roles/openshift-applier/tasks/process-helm.yml:40]: but, yes, this is probably why you split it into two before - let me think about this a bit more

Comment (jfilipcz) [reply roles/openshift-applier/tasks/process-helm.yml:40]: I was thinking about constructing entire helm command, including all flavours,  before the execution step - but decided to leave that aside as a future improvement task

**Comment by oybed** [comment]
@jfilipcz submitted PR against your PR branch to help with the robustness:
https://github.com/jfilipcz/openshift-applier/pull/1

File: roles/openshift-applier/tasks/process-helm.yml (hunk 1 above)
Comment (oybed) [reply roles/openshift-applier/tasks/process-helm.yml:34]: PR submitted: https://github.com/jfilipcz/openshift-applier/pull/1

File: roles/openshift-applier/tasks/process-helm.yml (hunk 2 above)
Comment (oybed) [reply roles/openshift-applier/tasks/process-helm.yml:40]: PR submitted: https://github.com/jfilipcz/openshift-applier/pull/1

**Comment by jfilipcz** [comment]
> @jfilipcz submitted PR against your PR branch to help with the robustness: [jfilipcz#1](https://github.com/jfilipcz/openshift-applier/pull/1)

Nice thing, I've merged the PR

**Review by oybed — APPROVED** [review]
LGTM

**Pull request merged**